chapters/functions.tex

```diff
@@ -1719,6 +1722,8 @@ \section{Function Inlining and Event Generation}\label{function-inlining-and-eve
 Late inlining is especially useful for differentiation and inversion of functions; for efficiency reasons it is then useful to replace all function calls with identical input arguments by one function call, before the inlining.
 \end{nonnormative}
 
+The default for late inlining is tool-specific, in particular tools may automatically delay inlining to differentiate or possible invert the function before inlining it.
+
```

Comment:
Above, the default was described after giving the meaning of `Inline = false`.  It would make sense to be consistent and describe the default after the meaning of `LateInline = false`.

Reply:
Replaced by new conversation.  Resolving.